server/src/main/java/org/candlepin/controller/Refresher.java

```diff
@@ -155,6 +155,7 @@ public void run() {
              * dirty, they will never get regenerated
              */
             Pool masterPool = poolManager.convertToMasterPool(subscription);
+            masterPool.setLocked(true);
```

Comment:
Let the CPM handle this; the fewer places this is done explicitly, the better.

Reply:
Removing it.